### DISCUSSION TITLE
Understanding NNN_COLORS

### DISCUSSION THREAD

**Discussion by SimonPlata** [jarun/nnn#1949] [Show and tell]
I struggled to determine how to configure the status bar color. Eventually, I discovered that the environment variable NNN_COLORS defines it. However, NNN_COLORS has four color fields; why is that? Well, it's because you can set a status bar color for each context. Each time you press the TAB key, you change your context, and the color of the status bar can help you stay aware of which context you're in.

For example, if you define:
~$ export NNN_COLORS='1234'

Then the status bar for context 1 will have color 1 (Red), context 2 will have color 2 (Green), and so on.

**Comment by jarun** [comment]
Your understanding is correct.
Thanks for sharing your experience!
